examples/usecases/mcp_github_to_slack_agent/mcp_agent.config.yaml

```
@@ -24,5 +24,11 @@ mcp:
       description: "Access GitHub API operations"
     slack:
       command: "npx"
-      args: ["-y", "@modelcontextprotocol/server-slack"]
-      description: "Access Slack API operations"
+      args: ["-y",
+        "slack-mcp-server@latest",
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 13 but found 8

(indentation)